src/snake_game/utils.cljs

```diff
@@ -1,74 +0,0 @@
-(ns snake-game.utils
-  (:require [reagent.core :as reagent :refer [atom]]))
-
-(def key-code->move
-  "Mapping from integer key code to direction vector corresponding to that key"
-  {38 [0 -1]
-   40 [0 1]
-   39 [1 0]
-   37 [-1 0]})
-
-(defn rand-free-position
-  "Fuction takes snake and board-size as arguments.,
-  and returns random position not colliding wit snake body"
-  [snake [x y]]
-  (let [snake-positions (into #{} (:body snake))
```

Comment:
I would even consider changing the name in `-set` to underline the idiomatic `remove` below